SocketIOClientSwift/SocketEngine.swift

```diff
@@ -350,8 +350,10 @@ public final class SocketEngine: NSObject, WebSocketDelegate {
     // We had packets waiting for send when we upgraded
     // Send them raw
     private func flushWaitingForPostToWebSocket() {
-        for msg in postWait {
-            ws?.writeString(msg)
+        if let ws = ws {
```

Comment:
This is a small optimization: there's no need to iterate through `postWait` if `ws` is `nil` anyway.